### ISSUE TITLE
The requested URL returned error: 404 Not Found

### ISSUE THREAD

**Issue by NikhilRadadiya** [appstract/laravel-opcache#79]
getting The requested URL returned error: 404 Not Found
Url: http://127.0.0.1/opcache-api/clear?key=eyJpdiI6InZiUG1WSEo2U2lXelgwNGZTR04rYWc9PSIsInZhbHVlIjoicUV2d0FBdkVVejMrS2NabzRRK3Q3UT09IiwibWFjIjoiNTEwYzY0ODFhZTM0Yjg4ZGNkYTA0M2I3ZmJmY2VjYzM2YmJkMmE0M2I2YTUyMjc0NWQ5YjZhOTMwOGQ1ODNiOCJ9

I'm using php 7.2 and laravel 5.6

getting above error with all opcahce commands.

tried setting APP_URL to localhost and 127.0.0.1 also.
tried php artisan config:clear also

**Comment by renfan** [comment]
change APP_URL in .env file.

**Comment by NikhilRadadiya** [comment]
@renfan did that already

**Comment by Luqman-Kodaq** [comment]
@NikhilRadadiya I had the same issue and I changed the CreatesRequest.php file in the appstract folder in the vendor:

CreatesRequest.php

**Before**
namespace Appstract\Opcache;

use Appstract\LushHttp\LushFacade as Lush;
use Illuminate\Support\Facades\Crypt as Crypt;

trait CreatesRequest
{
    /**
     * @param $url
     *
     * @return mixed
     */
    public function sendRequest($url)
    {
        return Lush::headers(config('opcache.headers'))->options(['verify_ssl' => config('opcache.verify_ssl')])->get(**config('opcache.url'**).'/opcache-api/'.$url,
            ['key' => Crypt::encrypt('opcache')]
        );
    }
}

**After**

<?php

namespace Appstract\Opcache;

use Appstract\LushHttp\LushFacade as Lush;
use Illuminate\Support\Facades\Crypt as Crypt;

trait CreatesRequest
{
    /**
     * @param $url
     *
     * @return mixed
     */
    public function sendRequest($url)
    {
        return Lush::headers(config('opcache.headers'))->options(['verify_ssl' => config('opcache.verify_ssl')])->get(**'http://127.0.0.1:8000'**.'/opcache-api/'.$url,
            ['key' => Crypt::encrypt('opcache')]
        );
    }
}

Basically, manually change the get() method's first parameter to your ip address. I don't know if its best practice but worked for me anyway.

**Comment by baoanhng** [comment]
The solution is a bit tricky if you are using `route:cache` with opcache `validate_timestamps` set to `false`.

First you need to run `php artisan route:clear`
Then clear opcache with `php artisan opcache:clear`
Finally we can cache the route again with `php artisan route:cache`
